src/components/selection_input/selection_input.ts

```diff
@@ -46,6 +46,7 @@ interface Props {
   onSelectionRemoved?: (index: number) => void;
```

Comment:
Doesn't work until closing/re-opening the side panel if we add a range that is disabled :x